### PR TITLE
feat(Util): add missing colors to docs

### PR DESCRIPTION
**Pull request by samsamson33**
This adds the following colors which were missing from the docs (as noted [here](https://github.com/discordjs/discord.js/issues/3840#issuecomment-591697024)):
- BLURPLE
- GREYPLE
- DARK_BUT_NOT_BLACK
- NOT_QUITE_BLACK

**Status**
- [x] Code changes have been tested against the Discord API, or there are no code changes
- [x] I know how to update typings and have done so, or typings don't need updating

**Semantic versioning classification:**  
- [ ] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [x] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by Tenpi** [comment]
Well based on #2729 these are some hidden memes/easter eggs that you can only find by digging into the source code :)

**Comment by samsamson33** [comment]
Oh! Well, they're all [official Discord Colors](https://discordapp.com/branding), so if that still stands then I imagine all of them would be Easter eggs. Still, I'll wait for a member to close this if they're intentionally undocumented.

**Comment by devsnek** [comment]
The documented colors are based on the color picker in the role ui in the app. The undocumented ones are based on discord's branding guidelines (which you linked to).

**Comment by samsamson33** [comment]
Right, so should the Discord banding colors be documented, or should they be kept undocumented as a secret of some sort?

File: src/util/Util.js [hunk 1]
```diff
@@ -420,6 +420,10 @@ class Util {
    * - `DARK_GREY`
    * - `LIGHT_GREY`
    * - `DARK_NAVY`
+   * - `BLURPLE`
+   * - `GREYPLE`
+   * - `DARK_BUT_NOT_BLACK`
+   * - `NOT_QUITE_BLACK`
```
Comment (NotSugden) [inline src/util/Util.js]: these should also be added to the `ColorResolvable` type in the typings

Comment (samsamson33) [reply src/util/Util.js]: Fixed!

**Pull request merged**